client/my-sites/plan-features/actions.jsx

```diff
@@ -23,15 +25,24 @@ const PlanFeaturesActions = ( {
 
 	if ( current ) {
 		upgradeButton = (
-			<Button className="plan-features__actions-button is-current" disabled>
+			<Button
+				className={ classNames( {
```

Comment:
We could extract this to a var above `if ( current ) {` like this:

```js

const classes = classNames( 'plan-features__actions-button', {
					'is-current': current
					'is-popular': popular
				} );
```

and then use it in both the button here and the button in the `} else if ( available...` case. :)